stdlib/urllib/error.pyi

```diff
@@ -11,6 +11,8 @@ class URLError(IOError):
 class HTTPError(URLError, addinfourl):
     @property
     def headers(self) -> Message: ...  # type: ignore[override]
```

Comment:
From my local experimentation, looks like this `type: ignore` is no longer needed (not sure if that's new in this PR, or if it's been true for a while)

```suggestion
    def headers(self) -> Message: ...
```